src/controller/ControllerPosition.cpp

```diff
@@ -35,6 +35,15 @@ namespace UKControllerPlugin {
             return this->type;
         }
 
+        bool ControllerPosition::HasTopdownAirfield(std::string icao) const

```

Comment:
const string on the method signature?

Reply:
Pass by value so doesn't make a tremendous difference :)